service/src/main/java/com/myhome/services/springdatajpa/MailSDJpaService.java

```diff
@@ -45,16 +65,59 @@ private boolean send(String emailTo, String subject, String message) {
 
   @Override
   public boolean sendPasswordRecoverCode(User user, String randomCode) {
-    String message = String.format(PASSWORD_RECOVER_MAIL_TEXT, user.getName(), randomCode);
-    boolean mailSent = send(user.getEmail(), PASSWORD_RECOVER_EMAIL_SUBJECT, message);
+    Map<String, Object> templateModel = new HashMap<>();
+    templateModel.put("username", user.getName());
+    templateModel.put("recoverCode", randomCode);
+    String passwordRecoverSubject = getLocalizedMessage("locale.EmailSubject.passwordRecover");
+    boolean mailSent = send(user.getEmail(), passwordRecoverSubject,
+        MailTemplatesNames.PASSWORD_RESET.name(),  templateModel);
```

Comment:
Are you sure that .name() is the thing that you want to call?
It returns the name of enum, so in case 'PASSWORD_RESET("something")' you will get "PASSWORD_NAME" instead of "something"